managers/asset-operations-manager.js

```diff
@@ -406,53 +398,60 @@ class AssetOperationsManager {
             hashFunctionId,
         );
 
-        let operationResult = await this.nodeApiService.getOperationResult(
+        let publishOperationResult = await this.nodeApiService.getOperationResult(
             endpoint,
             port,
             authToken,
             OPERATIONS.PUBLISH,
             maxNumberOfRetries,
             frequency,
-            operationId,
+            publishOperationId,
         );
 
-        if (operationResult.status === OPERATION_STATUSES.FAILED) {
+        if (publishOperationResult.status === OPERATION_STATUSES.FAILED) {
             return {
                 UAL,
                 assertionId: publicAssertionId,
-                operation: getOperationStatusObject(operationResult, operationId),
+                operation: getOperationStatusObject(publishOperationResult, publishOperationId),
             };
         }
 
-        operationId = await this.nodeApiService.localStore(
+        let localStoreOperationId = await this.nodeApiService.localStore(
             endpoint,
             port,
             authToken,
             assertions,
         );
 
-        operationResult = await this.nodeApiService.getOperationResult(
+        let localStoreOperationResult = await this.nodeApiService.getOperationResult(
             endpoint,
             port,
             authToken,
             OPERATIONS.LOCAL_STORE,
             maxNumberOfRetries,
             DEFAULT_GET_LOCAL_STORE_RESULT_FREQUENCY,
-            operationId,
+            localStoreOperationId,
         );
 
         stepHooks.afterHook({
             status: OPERATIONS_STEP_STATUS.CREATE_ASSET_COMPLETED,
             data: {
-                operationId,
-                operationResult,
+                localStoreOperationId,
+                localStoreOperationResult,
             },
         });
 
         return {
             UAL,
             publicAssertionId,
-            operation: getOperationStatusObject(operationResult, operationId),
+            operation: {
+                mintKnowledgeAssetReceipt: receipt,
```

Comment:
Let's make it the same as in `dkg.py`
```suggestion
                mintKnowledgeAsset: receipt,
```